src/positions.jl

```diff
@@ -36,7 +36,7 @@ function get_positions(symbol::String)::DataFrame
   return resdf
 end
 
-function get_positions()::DataFrame
+function get_all_positions()::DataFrame
```

Comment:
Don't need two function names to do the same thing

Reply:
k intent was to cater to ALL positions as well as a singular symbol. 

https://alpaca.markets/docs/api-references/trading-api/positions/

![image](https://github.com/dm13450/AlpacaMarkets.jl/assets/23480124/069bdbf6-f7ce-4b14-ba28-45fe2593c81f)

Reply:
Yeah I can see they do different things but we can overload one function to do both things, get_positions() and get_positions("BTCUSD") both work as intended